packages/api/src/server/analytics/service/analytics.service.ts

```diff
@@ -57,7 +57,7 @@ export class AnalyticsService {
     const savedAnalytics = await this.dataServices.activityStream.create(activityStream);
     try {
       const webhooks = await this.dataServices.webhook.getByAny({ propertyIdentifier, actions: action });
-      this.publishWebhookEvents(webhooks, activityStream);
+      if (webhooks.length !== 0) this.publishWebhookEvents(webhooks, activityStream);
```

Comment:
Can you please raise a different PR for this? Rest of the changes looks good to me.

Reply:
also, remove this from  the description "Provide Initial access sync issue resolved for webhook"

Reply:
Sure @arkaprovob

Reply:
Hi @arkaprovob , I've removed this.

Reply:
done